tests/test_emulate.py

```diff
@@ -30,8 +29,23 @@ def test_emulate_disasm(start_binary):
         '   0x400094             add    byte ptr [rax], al'
     ]
 
-    disasm_without_emu = [
-        ' ► 0x400080 <_start>      jmp    label                         <0x400083>',
+    disasm_with_emu_0x401000 = [
+        ' ► 0x401000 <_start>    jmp    label                      <label>',
```

Comment:
To keep green the Docker container and GitHub we need to test two outputs with different address. In Docker and my computer starts at 0x401000, GitHub starts at 0x400080. Added two expectations.

Reply:
Hmm instead of goin this attempt - with "either X or Y address" we should somehow force the binary to have **this exact addresses**. I'm pretty sure this can be done somehow, but I never got into this that deep.

Maybe with some linker script?

Reply:
Yup, we can pass in a flag to linker, so that it will put the `.text` section at a given address.

This way we can have a single address for both environments :)

Can you try it?

Shall be: `ld -Ttext 0x4000000 -o main main.o`
Probably here: https://github.com/pwndbg/pwndbg/blob/7690b6072210fecee732b9673d224118ff30b4b5/tests/binaries/makefile#L44

Reply:
Done in the commit f3baa38cb0400ac4ffe9ec009c5e6ceaa462c3c3

Your solution is the best! As we say in Mexico “eres chingon compa” or in Spain/Argentina “sos un crack”.

The testing command is `docker build -t pwndbg . && docker run --rm -it --cap-add=SYS_PTRACE pwndbg /bin/bash ./tests.sh`